flow/workflows/cdc_flow.go

```diff
@@ -181,6 +182,34 @@ func addCdcPropertiesSignalListener(
 	})
 }
 
+func reloadPeers(ctx workflow.Context, logger log.Logger, cfg *protos.FlowConnectionConfigs) error {
+	reloadPeersCtx := workflow.WithActivityOptions(ctx, workflow.ActivityOptions{
+		StartToCloseTimeout: 5 * time.Minute,
+	})
+
+	logger.Info(fmt.Sprintf("reloading source peer %s", cfg.Source.Name))
```

Comment:
```suggestion
	logger.Info("reloading source peer", slog.String("peerName", cfg.Source.Name))
```
annoying since datadog doesn't render as well but more structural is easier to query logs